.circleci/config.yml

```
@@ -8,7 +8,7 @@ jobs:
     # 'machine' executor runs Unit tests ~x1.5 faster, comparing to 'docker' executor
     # but the fastest is still ~x1.5-2 slower, comparing to Travis
     machine: true
-    parallelism: 4
+    parallelism: 3
```

```suggestion
    parallelism: 4
```

If we're adding focal at the same time that we're removing xenial, then let's leave this alone.

---

@blag Fixed...
Also yes my understanding is the same. It does the building and testing in the docker containers, and so the focal containers have python 3.8 so I believe this is all good. As the image is just what you run the essentially commands to pull up docker run in.